docs/modules/extensions/pages/inline-macro-processor.adoc

```diff
@@ -102,3 +110,28 @@ class CiteInlineMacroProcessor extends InlineMacroProcessor {
 ----
 
 With the SHORT format, the attributes are not parsed, and the 'target' that is passed to your macro is the value between the brackets ("brown79").
+
+== Positional attributes
+
+The first example here has shown how to access named attributes.
+But AsciiDoc also supports positional attributes where the meaning implicitly derived from the position in the attribute list.
+In that example the attribute `repo` might also be defined as the first attribute so that the inline macro might also be written as  `issue:333[asciidoctor/asciidoctorj]`.
```

Comment:
To avoid AsciiDoc looking for the `issue` macro here, it should be enclosed in literal monospace.

```
`+issue:333[asciidoctor/asciidoctorj]+`
```

Reply:
Thanks! I also updated the other places that had inline macros that were not supposed to be rendered.